packages/e2e/features/ui/components/authenticator/verify-user.feature

```diff
@@ -49,7 +49,7 @@ Feature: Verify User
     Then I see "Sign out"
     And I click the "Sign out" button
 
-  @angular @react @vue @react-nativee
+  @todo-angular @react @todo-vue @todo-react-native
```

Comment:
* CI is set up to ignore @todo-framework tests
* react native tests were not running for this suite, I have a follow-up asana ticket to address this (tldr, this feature reuses sign-in example app which is not mocked, due to how Detox handles mocking would need a separate app)